### PR TITLE
Updated LCM48/IMX582 docs

### PR DESCRIPTION
**Pull request by Erol444**

### CODE REVIEW

File: docs/source/pages/NG9096max.rst [hunk 1]
```diff
@@ -32,14 +32,17 @@ Dimensions and Weight
...
 .. image:: /_static/images/BW1093/oak-1-assembly.png
 
-Datasheet
-*********
+..
+    Datasheet
+    *********
 
-* `Datasheet <https://github.com/luxonis/depthai-hardware/blob/master/NG9096_OAK-1-Lite/Datasheet/OAK-1-Lite_Datasheet.pdf>`__
+    * `Datasheet <https://github.com/luxonis/depthai-hardware/blob/master/NG9096_OAK-1-Lite/Datasheet/OAK-1-Lite_Datasheet.pdf>`__
```
Comment (cafemoloko) [inline docs/source/pages/NG9096max.rst:39]: Link to OAK-1-Lite datasheet.

**Review by cafemoloko — APPROVED** [review]
LGTM.

File: docs/source/pages/NG9096max.rst (hunk 1 above)
Comment (Erol444) [reply docs/source/pages/NG9096max.rst:39]: do we have OAK-1-Max datasheet?

Comment (Erol444) [reply docs/source/pages/NG9096max.rst:39]: (as this is currently commented out)

**Pull request merged**